charts/generic-prometheus-alerts/values.yaml

```diff
@@ -70,18 +70,18 @@ additionalRuleLabels: {}
 # Set the alert threshold for total number of SQS on the queue waiting to be processed.
 # sqsAlertsTotalMessagesThreshold: 100
 
-# To monitor AWS SNS Topic queues, add the name of the topic in a list.
+# To monitor AWS SNS Topic queues, add the topic names in a list.
 # snsAlertsTopicNames:
 #   topic-name-id: "Topic nice display name"
 #   topic-name-2-id: "Topic nice display name 2"
 
 # Set the alert threshold for no SNS messages published to topic (minutes).
 # snsAlertsNoMessagesThresholdMinutes: 30
 
-# To monitor AWS Elasticache, add the name of the clusterId in a list.
+# To monitor AWS Elasticache, add the cluster node names in a list as follows.
```

Comment:
Trying to make it a bit clearer that we are listing the NODE names within the cluster and not just the cluster name.  I think the `elastiCacheAlertsClusterIds` is a bit misleading really... guess I could change it and do a major release?